Documentation/articles/getting_started/getting_started.md

```diff
@@ -1,10 +1,30 @@
 # Getting Started
 
-This section walks you through the basics of MonoGame and help you create your first game.
+This section walks you through the basics of MonoGame and helps you create your first game.
+
+First, select the toolset and operating system you will be working with to create your first MonoGame project and then continue your journey to understand the basic layout of a MonoGame project.
+
+By the end of this tutorial set, you will have a working project to start building from for your target platform and be ready to tackle your next steps.
+
+## Creating a new Project
+
+- 3.8
+
+  - [Visual Studio](3.8/1_creating_a_new_project_vs.md)
+  - [Visual Studio for Mac](3.8/1_creating_a_new_project_vsm.md)
+  - [DotNet core CLI](3.8/1_creating_a_new_project_netcore.md)
+
+- 3.7
+
+  - [Visual Studio](3.7/1_creating_a_new_project_vs.md)
+  - [MonoDevelop / Xamarin Studio](3.7/1_creating_a_new_project_md.md)
+
+## Building your game
 
-- [Creating a New Project](1_creating_a_new_project.md)
-  - [Visual Studio](1_creating_a_new_project_vs.md)
-  - [MonoDevelop / Xamarin Studio](1_creating_a_new_project_md.md)
 - [Understanding the Code](2_understanding_the_code.md)
 - [Adding Content](3_adding_content.md)
 - [Adding Basic Code](4_adding_basic_code.md)
+
+## [Migrating from MonoGame 3.7 to 3.8?](/migrate38.md)
+
+If you currently have a MonoGame 3.7 project and wish to upgrade it to MonoGame 3.8, please check our handy [Upgrade Guide](/migrate38.md).
```

Comment:
It's ok to have a migration from 3.7 guide for the next few versions of MG since we're making the big move from installers to NuGet.  That said... these docs don't seem to exist... the link doesn't work.

Reply:
Needs to be `~/articles/migrate38.md`: https://docs.monogame.net/articles/migrate38.html

Reply:
Fixed